common/stacks/src/test/scala/weco/api/stacks/services/SierraServiceTest.scala

```diff
@@ -239,216 +239,174 @@ class SierraServiceTest
-                  |  "total": 1,
-                  |  "start": 0,
-                  |  "entries": [
-                  |    {
-                  |      "id": "$item",
-                  |      "deletedDate": "2001-01-01",
-                  |      "deleted": false,
-                  |      "suppressed": true,
-                  |      "fixedFields": {
-                  |        "88": {"label": "STATUS", "value": "-", "display": "Available"}
-                  |      },
-                  |      "holdCount": 0
-                  |    }
-                  |  ]
-                  |}
-                  |""".stripMargin
-              )
+          HttpResponse(
+            entity = HttpEntity(
+              contentType = ContentTypes.`application/json`,
+              s"""
+                 |{
+                 |  "total": 0,
+                 |  "start": 0,
+                 |  "entries": []
+                 |}
+                 |""".stripMargin
+            )
+          )
+        ),
+        (
+          HttpRequest(
+            uri = Uri(
+              f"http://sierra:1234/v5/items?id=$item&fields=deleted,fixedFields,holdCount,suppressed"
+            )
+          ),
+          HttpResponse(
+            entity = HttpEntity(
+              contentType = ContentTypes.`application/json`,
+              f"""
+                 |{
+                 |  "total": 1,
+                 |  "start": 0,
+                 |  "entries": [
+                 |    {
+                 |      "id": "$item",
+                 |      "deletedDate": "2001-01-01",
+                 |      "deleted": false,
+                 |      "suppressed": true,
+                 |      "fixedFields": {
+                 |        "88": {"label": "STATUS", "value": "-", "display": "Available"}
+                 |      },
+                 |      "holdCount": 0
+                 |    }
+                 |  ]
+                 |}
+                 |""".stripMargin
             )
           )
         )
+      )
 
-        withMaterializer { implicit mat =>
-          val service = SierraService(
-            new MemoryHttpClient(responses) with HttpGet with HttpPost {
-              override val baseUri: Uri = Uri("http://sierra:1234")
-            }
-          )
+      withMaterializer { implicit mat =>
+        val service = SierraService(
+          new MemoryHttpClient(responses) with HttpGet with HttpPost {
+            override val baseUri: Uri = Uri("http://sierra:1234")
+          }
+        )
 
-          val future = service.placeHold(
-            patron = patron,
-            sourceIdentifier = sourceIdentifier
-          )
+        val future = service.placeHold(
+          patron = patron,
+          sourceIdentifier = sourceIdentifier
+        )
 
-          whenReady(future) {
-            _.left.value shouldBe HoldRejected.ItemCannotBeRequested
-          }
+        whenReady(future) {
+          _.left.value shouldBe HoldRejected.ItemCannotBeRequested
```

Comment:
I'm sort of trusting you haven't fiddled with this file too much, because the indentation makes it harder to tell.

Reply:
I went through and de-jiggled it - I think a curly bracket had tried to make a run for it.